content/api/resources/ledgers/object.mdx

```diff
@@ -54,7 +54,7 @@ When Horizon returns information about a ledger, it uses the following format:
   - The reserve the network uses when calculating an account’s minimum balance.
 - max_tx_set_size
   - number
-  - The maximum number of transactions validators have agreed to process in a given ledger.
+  - The maximum number of transaction operations validators have agreed to process in a given ledger.
```

Comment:
```suggestion
  - The maximum number of operations validators have agreed to process in a given ledger.  Since Protocol 11, ledger capacity has been measured in operations rather than transactions.  For more info on that decision, see [CAP-5](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0005.md#repurposing-ledgerheadermaxtxsetsize).
```